docs/src/main/sphinx/security/ranger-access-control.md

```diff
@@ -1,6 +1,9 @@
-# Apache Ranger access control
+# Ranger access control
```

Comment:
Leave this one. It's the name of the product, so it should use the proper naming.

Reply:
I changed it because its inconsistent with all the connectors ..and if we rename those it will be tricky .. for example LF Delta Lake connector .. and Apache Iceberg connector.. 

But I am also happy to be inconsistent since security is kinda separate from connectors.

Reply:
We decided to run with consistency..